test/api-test/api/metadata/jobcodes/{id}/remove/delete-pre-request.js

```diff
@@ -0,0 +1,2 @@
+let id = pm.request.url.variables.indexOf('id');
+pm.request.url.variables.idx(id).value = "21";
```

Comment:
You don't need this file because the get-pre-request.js takes care of adding the id for /remove and /update.